File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsAboutLicensesFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsAboutLicensesFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsAboutLicensesFragment().getTitleResourceId()).isEqualTo(R.string.licenses);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsAboutLicensesFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_about_licenses);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutLicensesFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsAboutLicensesFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsAboutLicensesFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.licenses)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsAboutLicensesFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings_about_licenses)
	}
}
